### PR TITLE
Added Exif code examples

### PR DESCRIPTION
**Pull request by radarhere**
Suggestion for https://github.com/python-pillow/Pillow/pull/6930

### CODE REVIEW

File: src/PIL/Image.py [hunk 1]
```diff
@@ -3607,18 +3607,36 @@ def _apply_env_variables(env=None):
...
+    ``ExifTags.IFD.Interop`` and ``ExifTags.IFD.IFD1``.
+
+    :py:mod:`~PIL.ExifTags` also has enum classes to provide names for data::
 
-    Both root Exif and child IFD objects support dict interface and can be
-    indexed by int values that are available as enum members of
-    :py:class:`~PIL.ExifTags.Base`, :py:class:`~PIL.ExifTags.GPS`, and
-    :py:class:`~PIL.ExifTags.Interop`.
+      print(exif[ExifTags.Base.Software])  # PIL
+      print(gps_ifd[ExifTags.GPS.GPSDateStamp])  # '1999:99:99 99:99:99'
```
Comment (greatvovan) [inline src/PIL/Image.py:3639]: In my experience, this entry contains date only, with no time (it in a separate entry), but the comment suggests differently. Maybe it's up to vendor?

File: src/PIL/Image.py [hunk 2]
```diff
@@ -3607,18 +3607,36 @@ def _apply_env_variables(env=None):
...
+      im = Image.open("exif.png")
+      exif = im.getexif()  # Returns an instance of this class
+
+    Information can be read and written, iterated over or deleted::
+
+      print(exif[274])  # 1
+      exif[274] = 2
+      for k, v in exif.items():
+        print("Tag", k, "Value", v)  # Tag 274 Value 2
+      del exif[274]
```
Comment (greatvovan) [inline src/PIL/Image.py:3622]: We probably want to show that the tag codes are integers, and I really like the conciseness of the example, but realistically, I think, no one would use random-looking numbers in real application code. Isn't it better to show it with an enum value?

```python
from PIL.ExifTags import Base
print(exif[Base.Orientation])  # 1
exif[Base.Orientation] = 2
for k, v in exif.items():
  print("Tag", Base(k).name, "Value", v)  # Tag Orientation Value 2
del exif[Base.Orientation]
```

**Review by greatvovan — COMMENTED** [review]
Thank you for the improvement @radarhere! I like the example-based approach, code illustrates it better than words.

File: src/PIL/Image.py (hunk 1 above)
Comment (radarhere) [reply src/PIL/Image.py:3639]: Here is the image I was using - https://github.com/python-pillow/Pillow/blob/main/Tests/images/exif_gps.jpg

File: src/PIL/Image.py (hunk 2 above)
Comment (radarhere) [reply src/PIL/Image.py:3622]: While the ExifTags enums are more helpful to humans, they are not as general - I expect that there are many tags that are not included in ExifTags, and I wouldn't want to users to think they are limited to just what we have listed.

I would hope that my last example in this docstring demonstrates the use of ExifTags sufficiently.

**Pull request merged**